pkg/storkctl/clusterpair.go

```diff
@@ -73,3 +76,38 @@ func clusterPairPrinter(clusterPairList *storkv1.ClusterPairList, writer io.Writ
 	}
 	return nil
 }
+
+func newGenerateClusterPairCommand(cmdFactory Factory) *cobra.Command {
+	generateClusterPairCommand := &cobra.Command{
+		Use:   clusterPairSubcommand,
+		Short: "Generate a spec to be used for cluster pairing from a remote cluster",
+		Run: func(c *cobra.Command, args []string) {
+			config, err := cmdFactory.RawConfig()
+			if err != nil {
+				handleError(err)
+			}
+
+			clusterPair := &storkv1.ClusterPair{
+				TypeMeta: meta.TypeMeta{
+					Kind:       reflect.TypeOf(storkv1.ClusterPair{}).Name(),
+					APIVersion: storkv1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: meta.ObjectMeta{
+					Name: "<insert_name_here>",
```

Comment:
We can add a prefix based on the `config` here so that one can generate difficult cluster pair specs per context. config has CurrentContext which we could use. for `cluster-pair-ctx-<current-context>`

Reply:
There needs to be one context per cluster pair. If you use `storkctl generate clusterpair --context` it'll generate it for that context